crates/cxx-qt-gen/src/generator/rust/constructor.rs

```diff
@@ -115,14 +110,14 @@ fn generate_arguments_struct(
 fn generate_arguments_initialization(
     struct_name: &Ident,
     instance_name: Ident,
-    argument_list: &Option<Vec<Type>>,
+    argument_list: &[Type],
 ) -> Expr {
     let init_arguments = extract_arguments_from_tuple(argument_list, instance_name);
     println!("init_arguments: {:?}", init_arguments);
```

Comment:
Can we remove this print line ?

When building it causes `[build] init_arguments: []` to appear etc

Reply:
Oh, yes definitely!